README.md

```diff
@@ -16,12 +16,79 @@ goose
 <a href="#open-source">Open Source</a>
 </p>
 
-`goose` assists in solving a wide range of programming and operational tasks. It is a live virtual developer you can interact with, guide, and learn from.
+**Welcome to `goose`, the Open-Source, Extensible AI Coding Assistant!** 
 
-To solve problems, `goose` breaks down instructions into sequences of tasks and carries them out using tools. Its ability to connect its changes with real outcomes (e.g. errors) and course correct is its most powerful and exciting feature. `goose` is free open source software and is built to be extensible and customizable.
+Unlike traditional coding tools, `goose` leverages advanced natural language processing to interpret complex commands into actionable, executable sequences. This allows for a coding assistant that not only writes code but also understands the intent behind your commands, enabling it to manage broader project scopes autonomously.
+
+```
+TODO: quick install instructions will go here in a callout
+```
+
+`TODO: slow video down and center it on the page`
 
 ![goose_demo](https://github.com/user-attachments/assets/0794eaba-97ab-40ef-af64-6fc7f68eb8e2)
 
+## What Sets Goose Apart?
+
+* **Adaptability**: `goose` excels in its ability to adapt to changes in code in real-time, making necessary adjustments as errors occur. This mimics a human developer's approach to coding, iteratively refining the work until perfection is achieved.
+* **Autonomy**: While tools like Copilot provide line-by-line code suggestions based on comments or partial code snippets, goose can take full command descriptions to manage entire projects with little to no ongoing input from developers.
+* **Customization and Extensibility**: Open-source and fully customizable, goose is built to support extensive user-defined modifications, unlike the mostly closed systems of Copilot and Cursor. Developers can tailor `goose` extensively to fit specific workflows, integrate with various toolkits, and enhance functionality through community contributions.
+* **Privacy and Security**: Operating locally on your device, `goose` ensures that your proprietary code and sensitive data are never sent to the cloud.
+
+## Features and capabilities
+`#TODO: include table (?) but add some more descriptions on goose's capabilities`
+
+|  | goose | aider | copilot | cursor |
```

Comment:
nice! Maybe we don't need the whole list of LLMs?

Seems like you'll add, but for capabilities are you thinking of the core toolkits like screen capture, gh cli, etc..?

Reply:
Yeah agreed! I wanted to share what I came up with to start with the intent of getting people to think about what to add / remove. I also think that it's not to goose's advantage to show aider has more LLM connectivity.

I also thought maybe if our team saw this they'd want to maybe add features for goose that we don't have but others do?